botocore/client.py

```diff
@@ -651,44 +795,59 @@ class BaseClient(object):
     # we need the reverse mapping here.
     _PY_TO_OP_NAME = {}
 
-    def __init__(self, serializer, endpoint, response_parser,
-                 event_emitter, request_signer, service_model, loader,
-                 client_config, partition, exceptions_factory):
+    def __init__(
+        self,
+        serializer,
+        endpoint,
+        response_parser,
+        event_emitter,
+        request_signer,
+        service_model,
+        loader,
+        client_config,
+        partition,
+        exceptions_factory,
+    ):
         self._serializer = serializer
         self._endpoint = endpoint
         self._response_parser = response_parser
         self._request_signer = request_signer
         self._cache = {}
         self._loader = loader
         self._client_config = client_config
-        self.meta = ClientMeta(event_emitter, self._client_config,
-                               endpoint.host, service_model,
-                               self._PY_TO_OP_NAME, partition)
+        self.meta = ClientMeta(
+            event_emitter,
+            self._client_config,
+            endpoint.host,
+            service_model,
+            self._PY_TO_OP_NAME,
+            partition,
+        )
         self._exceptions_factory = exceptions_factory
         self._exceptions = None
         self._register_handlers()
 
     def __getattr__(self, item):
-        event_name = 'getattr.%s.%s' % (
-            self._service_model.service_id.hyphenize(), item
+        event_name = 'getattr.{}.{}'.format(
```

Comment:
Interesting that it went to `format` here. Maybe because the service_id line is too long?